src/collect.ts

```diff
@@ -0,0 +1,248 @@
+import checker from 'license-checker';
+import { exit } from 'process';
+import fs from 'fs';
+import path from 'path';
+import { CachedFetch } from './cached-fetch';
+
+export interface ErrorOnPackage {
+  name: string;
+  error?: string;
+}
+
+/**
+ * Object of different options used in {@link collectLicenses}
+ */
+export interface Options {
```

Comment:
Interfaces here could be dumped in separate files would make the file length shorter.

Reply:
Agreed